File: Java/Lista_02(Estrutura_Condicional)/Lanchonete.java
import java.util.Locale;
import java.util.Scanner;

public class Lanchonete {

	public static void main(String[] args) {

		Scanner s = new Scanner(System.in);
		
		System.out.print("Codigo do produto comprado: ");
		int codigo = s.nextInt();
		
		double preco;
		
		switch(codigo) {
		
		case 1:
			preco = 5.00;
			break;
		case 2:
			preco = 3.50;
			break;
		case 3:
			preco = 4.80;
			break;
		case 4:
			preco = 8.90;
		case 5:
			preco = 7.32;
			break;
		default:
			preco = 0.00;
			break;
			
		}
		
		System.out.print("Quantidade comprada: ");
		int quantidade = s.nextInt();
		
		double aPagar = preco * quantidade;
		
		Locale.setDefault(Locale.US);
		System.out.printf("Valor a pagar = R$ %.2f", aPagar);
		
		s.close();
		
	}

}

File: Python/Lista_02(Estrutura_Condicional)/Lanchonete.py
cod = (input("Codigo do produto comprado: "))

if cod == '1':
    preco = 5.00
elif cod == '2':
    preco = 3.50
elif cod == '3':
    preco = 4.80
elif cod == '4':
    preco = 8.90
elif cod == '5':
    preco = 7.32
else:
    preco = 0.00

quant = int(input("Quantidade comprada: "))

aPagar = preco * quant

print(f"Valor a pagar: R$ {aPagar:.2f}")